fig/backends/chronicle/README.md

```diff
@@ -16,6 +16,8 @@ backends=CHRONICLE
 # Uncomment to provide Google security key. Alternatively, use GOOGLE_SECURITY_KEY variable
 #security_key =
 
+# Uncomment to provide Chronicle region (us, europe, asia-southeast1). Alternatively, use CHRONICLE_REGION variable
```

Comment:
```suggestion
# Uncomment to provide Chronicle region (us, europe, asia-southeast1). Alternatively, use CHRONICLE_REGION environment variable
```

Reply:
Attempting to clarify that `CHRONICLE_REGION` is not a variable defined in config/config.ini